experimental/traffic-portal/src/app/core/statuses/status-details/status-details.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { StatusDetailsComponent } from './status-details.component';
+import { RouterModule } from '@angular/router';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import {MatGridListModule} from '@angular/material/grid-list';
+import { MatCardModule } from '@angular/material/card';
+import { SharedModule } from 'src/app/shared/shared.module';
+import { MatButtonModule } from '@angular/material/button';
+
+const StatusDetailRouting = RouterModule.forChild([
+  {
+    path: '',
+    component: StatusDetailsComponent
+  }
+]);
+
+@NgModule({
+  declarations: [
+    StatusDetailsComponent
+  ],
+  imports: [
+    CommonModule,
+    StatusDetailRouting,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatGridListModule,
+    MatCardModule,
+    MatButtonModule,
+    SharedModule
+  ]
+})
+export class StatusDetailsModule { }
```

Comment:
Why did you decide to make a whole module for each single component?

Reply:
Thought of lazyload the component so that in long run, application will be more scalable and loosely coupled.

Reply:
Thought of lazyload the component so that in long run, application will be more scalable and loosely coupled.

Reply:
I think that's pretty overkill, to make a different module for every single view. At best, it's premature optimization, and at worst it's detrimental because of increased loading times and significantly increased build times.